Marlin/Configuration.h

```diff
@@ -646,6 +646,8 @@ const bool Z_MIN_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the l
   #define EEPROM_CHITCHAT // Please keep turned on if you can.
 #endif
 
+//#define HOST_KEEPALIVE_FEATURE // Send messages to the host when busy with long operations
```

Comment:
Until there is at least one host supporting this, it makes no sense to activate it by default.
And then we have to find out, if this confuses any other hosts.